core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.html

```diff
@@ -27,10 +27,28 @@ <h4>{{ title$ | async }}</h4>
 </div>
 <div class="iframe-wrapper">
     <iframe
+        #iframe
         [src]="iframeUrl$ | async | safeUrl"
+        [title]="host"
         data-testId="iframe"
-        frameborder="0"
+        style="border: none"
         width="100%"
         height="100%"
         title="Editor Page"></iframe>
 </div>
+<p-dialog
+    #dialog
+    [(visible)]="visible"
+    [style]="{ height: '90vh', width: '90vw' }"
+    [modal]="true"
+    [header]="header"
+    [draggable]="false"
+    [resizable]="false">
+    <iframe
+        #dialogIframe
+        (load)="onIframeLoad($event)"
```

Comment:
We need to add a spiner in the dialog while this iframe loads... I know right now feels fast, bit in real life takes a few seconds.